Tests/MapboxSearchTests/Legacy/SearchResponseTests.swift

```diff
@@ -98,20 +98,4 @@ class SearchResponseTests: XCTestCase {
         let processedResponse = try response.process().get()
         XCTAssertEqual(processedResponse.suggestions.map({ $0.id }), expectedResults.map({ $0.id }))
     }
-    
-    func testSuccessResponseWithUnsupportedType_UserRecord() throws {
```

Comment:
It might be useful to leave the test but maybe change the expectation.

Reply:
I'll add additional tests here with `Brand`-type support.